integrations/Klaviyo/browser.js

```diff
@@ -57,20 +57,26 @@ class Klaviyo {
       return;
     }
     let payload = {
-      $id: message.userId || message.anonymousId,
+      $id: message.userId || message.context.userId || message.anonymousId,
```

Comment:
let's use get for this since we now have the lib?

Reply:
yes